ui/app/models/cluster.js

```diff
@@ -49,47 +49,17 @@ export default DS.Model.extend({
 
   anyReplicationEnabled: or('{dr,performance}.replicationEnabled'),
 
-  stateDisplay(state) {
-    if (!state) {
-      return null;
+  modeState: computed(
+    'dr.mode',
+    'performance.mode',
+    'replicationMode',
+    'dr.state',
+    'performance.state',
+    function() {
+      const mode = this.replicationMode;
+      return this.get(`${mode}.state`);
     }
-    const defaultDisp = 'Synced';
-    const displays = {
-      'stream-wals': 'Streaming',
-      'merkle-diff': 'Determining sync status',
-      'merkle-sync': 'Syncing',
-    };
-
-    return displays[state] || defaultDisp;
-  },
-
-  drStateDisplay: computed('dr.state', function() {
-    return this.stateDisplay(this.get('dr.state'));
-  }),
-
-  performanceStateDisplay: computed('performance.state', function() {
-    return this.stateDisplay(this.get('performance.state'));
-  }),
-
-  stateGlyph(state) {
-    const glyph = 'check-circle-outline';
-
-    const glyphs = {
```

Comment:
these are also now reflected in the `CLUSTER_STATES` map: https://github.com/hashicorp/vault/pull/8787/files#diff-b8f0db91f050a7646bbb0d17521230a2R20